sample-compose/app/src/androidTest/java/co/nimblehq/sample/compose/test/CoroutineTestRule.kt

```diff
@@ -0,0 +1,32 @@
+package co.nimblehq.sample.compose.test
+
+import co.nimblehq.sample.compose.util.DispatchersProvider
+import kotlinx.coroutines.*
+import kotlinx.coroutines.test.*
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class CoroutineTestRule(
+    private val testDispatcher: TestDispatcher = UnconfinedTestDispatcher()
+) : TestWatcher() {
+
+    override fun starting(description: Description?) {
+        super.starting(description)
+        Dispatchers.setMain(testDispatcher)
```

Comment:
Assuming you got this from the code example [here](https://developer.android.com/kotlin/coroutines/test
) 
```suggestion
        Dispatchers.setMain(testDispatcher)
```

Reply:
@Tuubz Good point. The default implementation is empty so `super` is unnecessary 👍 

![image](https://user-images.githubusercontent.com/16315358/218327917-3bd46e7d-4188-4634-b06a-acbc64c1d22a.png)

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/384/commits/d0a4ce710d805742b5160d44b28f657ed6b8f854

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/384/commits/d0a4ce710d805742b5160d44b28f657ed6b8f854